src/config/ssd.md

```diff
@@ -0,0 +1,153 @@
+# Solid State Drives
+
+Post installation, you will need to enable TRIM for solid state drives. You can
+check which devices allow TRIM by running:
+
+```
+$ lsblk --discard
+```
+
+If DISC-GRAN (discard granularity) and DISC-MAX (discard max bytes) columns are
+non-zero, that means the block device has TRIM support. If your solid state
+drive partition does not show TRIM support, please verify that you chose a file
+system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. For example, if your
+/ directory is on a SSD, run the following command to run TRIM one-shot:
+
+> Note: To run fstrim on F2FS file systems, you need to be running kernel 4.19
+> or greater.
+
+```
+# fstrim /
+```
+
+We can automate running TRIM by using cron or by adding the `discard` option to
+fstab.
+
+## Periodic TRIM with Cron
+
+Add the following lines to `/etc/cron.daily/fstrim`.
+
+```
+#!/bin/sh
+
+fstrim /
+```
+
+Finally, set the execution bit for the script:
+
+```
+# chmod u+x /etc/cron.daily/fstrim
+```
+
+## Continuous TRIM with Fstab discard
+
+> Note: You can use either continuous or periodic TRIM, but usage of continuous
+> TRIM is discouraged if you have an SSD that doesn't handle NCQ correctly.
+> Refer to the
+> [blacklist](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/ata/libata-core.c#n4522).
+
+Edit `/etc/fstab` and add the `discard` option to block devices that need TRIM.
+
+For example, if `/dev/sda1` was an SSD partition, formatted as ext4, and mounted
+to `/` with discard options, add the `discard` option:
+
+```
+/dev/sda1  /           ext4  defaults,discard   0  1
+```
+
+## LVM
+
+To enable TRIM for LVM's commands (`lvremove`, `lvreduce`, etc) commands, we
+must open `/etc/lvm/lvm.conf`, uncomment the `issue_discards` option, and set it
+to `1`:
+
+```
+issue_discards=1
+```
+
+## LUKS
+
+> Note: Before enabling discard for your LUKS partition, please be aware of the
+> [security
+> implications](https://wiki.archlinux.org/index.php/Dm-crypt/Specialties#Discard/TRIM_support_for_solid_state_drives_(SSD)).
+
+To open an encrypted LUKS device and allow discards to passthrough, open the
+device with the `--allow-discards` option.
+
+```
+# cryptsetup luksOpen --allow-discards /dev/sdaX luks
+```
+
+### Non-root devices
+
+Edit `/etc/crypttab` and set the `discard` option for devices on the SSD. For
+example if you have a LUKS device with the name `externaldrive1`, device
+`/dev/sdb2`, and `none` password, add the `discard` option:
+
+```
+externaldrive1  /dev/sdb2   none    luks,discard
+```
+
+### Root devices
+
+If your root-device is on LUKS, add `rd.luks.allow-discards` to
+`CMDLINE_LINUX_DEFAULT`. We can do this for grub by editing `/etc/default/grub`
+and setting:
+
+```
+GRUB_CMDLINE_LINUX_DEFAULT="rd.luks.allow-discards"
+```
+
+### Verifying Configuration
```

Comment:
"configuration", assuming we aren't using title case for headings (and if we are, other heads will need to be changed to use title case where appropriate).

Reply:
I am only using title case for the first heading (`Solid State Devices`)